diff --git a/src/connections/shift_reg_gpio.py b/src/connections/shift_reg_gpio.py
index ddfb4d9..4b3e0e5 100644
--- a/src/connections/shift_reg_gpio.py
+++ b/src/connections/shift_reg_gpio.py
@@ -1,4 +1,4 @@
-import RPi.GPIO as GPIO
+import connections.gpio_dummy as GPIO
 import time
 import logging
 from collections import namedtuple
diff --git a/src/controllable_objects/test_turn_all_on.py b/src/controllable_objects/test_turn_all_on.py
index a51acb8..6e62d88 100644
--- a/src/controllable_objects/test_turn_all_on.py
+++ b/src/controllable_objects/test_turn_all_on.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-import RPi.GPIO as GPIO
+import connections.gpio_dummy as GPIO
 import time
 
 from connections.shift_reg_gpio import ShiftRegGPIO
diff --git a/src/controller.py b/src/controller.py
index 353bf9b..f53f35f 100644
--- a/src/controller.py
+++ b/src/controller.py
@@ -1,4 +1,4 @@
-import RPi.GPIO as GPIO
+import connections.gpio_dummy as GPIO
 import logging
 import os.path
 
diff --git a/tests/manual/shift_reg/test_pins_manual.py b/tests/manual/shift_reg/test_pins_manual.py
index 8c50732..1d879c7 100644
--- a/tests/manual/shift_reg/test_pins_manual.py
+++ b/tests/manual/shift_reg/test_pins_manual.py
@@ -1,4 +1,4 @@
-import RPi.GPIO as GPIO
+import connections.gpio_dummy as GPIO
 import time
 
 from utils.get_user_answer import get_user_answer
diff --git a/tests/unittests/shift_reg/test_shift_reg_gpio.py b/tests/unittests/shift_reg/test_shift_reg_gpio.py
index 916b111..0076231 100644
--- a/tests/unittests/shift_reg/test_shift_reg_gpio.py
+++ b/tests/unittests/shift_reg/test_shift_reg_gpio.py
@@ -1,4 +1,4 @@
-import RPi.GPIO as GPIO
+import connections.gpio_dummy as GPIO
 import unittest
 
 from connections.shift_reg_gpio import ShiftRegGPIO
